### PR TITLE
CMake: Partial install

### PR DESCRIPTION
**Pull request by chfast**
Install commands required for Hera install to work.

### CODE REVIEW

File: libs/evm2wasm/CMakeLists.txt [hunk 1]
```diff
@@ -15,5 +15,11 @@ if(BUILD_SHARED_LIBS)
   set_target_properties(libevm2wasm PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
 endif()
 
-target_include_directories(libevm2wasm PUBLIC ${include_dir})
-target_link_libraries(libevm2wasm PRIVATE binaryen::binaryen Threads::Threads fmt::fmt)
\ No newline at end of file
+target_include_directories(libevm2wasm PUBLIC $<BUILD_INTERFACE:${include_dir}>$<INSTALL_INTERFACE:include>)
```
Comment (axic) [inline libs/evm2wasm/CMakeLists.txt]: There's no space between the build/install_interface parts, is that intentional?

Comment (chfast) [reply libs/evm2wasm/CMakeLists.txt]: This probably no strictly required but it is a common pattern I copy.

These are "generator expressions" and in build context this evaluates to `${include_dir}` (second part evaluates to nothing) and in install context this evaluates to `install` (first part evaluates to nothing).

File: CMakeLists.txt [hunk 2]
```diff
@@ -25,3 +27,11 @@ option(EVM2WASM_TOOLS ON)
 if(EVM2WASM_TOOLS)
     add_subdirectory(tools)
 endif()
+
+install(DIRECTORY include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+
+install(
+    EXPORT evm2wasmTargets
```
Comment (axic) [inline CMakeLists.txt]: Is this a cmake level name? Can be used by externalproject to refer to stuff?

Comment (chfast) [reply CMakeLists.txt]: This describes a "CMake package" but unfortunately this cannot be used in ExternalProject. This actually what Hunter workarounds.

**Pull request merged**